src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandler.java

```diff
@@ -71,23 +71,23 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             return response;
         }
 
-        caseData = ccdCallbackMapService.handleCcdCallbackMap(callbackMap, caseData);
-
-        handleSetAsideRefusedSor(caseData);
+        boolean isSetAsideRefusedSor = isSetAsideRefusedSor(postHearing);
```

Comment:
Any reason for this to be above the clearing of fields and ccd method?

Reply:
Yes - I need to know if that is true before the information is deleted forever. It is referenced below before the return, but after the ccd service is called.